### ISSUE TITLE
NuGet Packet does not work under Linux and MacOS

### ISSUE THREAD

**Issue by thinking-aloud** [BitMiracle/libtiff.net#43]
Hi,
I tried to use LibTIFF under Linux and OSX. The NuGet packet installs fine, but in both cases I get the exact same error message:
`Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'BitMiracle.LibTiff.NetStandard, Version=2.4.583.0, Culture=neutral, PublicKeyToken=53879b3e20e7a7d6'. The system cannot find the file specified.`

**Comment by Bobrovsky** [comment]
If the package install fine then when do you get the exception? Who throws it?

**Comment by thinking-aloud** [comment]
During runtime, this is all the information I get.

```
$ dotnet bin/Debug/netcoreapp1.1/dotnet.dll

Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'BitMiracle.LibTiff.NetStandard, Version=2.4.583.0, Culture=neutral, PublicKeyToken=53879b3e20e7a7d6'. The system cannot find the file specified.

   at dotnet.GeoTIFF.libTiffReader..ctor(String fileName)
   at dotnet.Program.Main(String[] args)
[1]    6251 abort      dotnet bin/Debug/netcoreapp1.1/dotnet.dll
```

**Comment by arturocandela** [comment]
Hi. I've have the same problem in Windows 10 using the dotnet console sdk and nuget. It seems that is a bug introduced for windows in the version 2.4.583 because I've forced the version 2.4.582 and It works perfect on Windows. Today if I have time I will check that in Mac:

Output from Windows powershell

`Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'BitMiracle.LibTiff.NetStanda
ersion=2.4.583.0, Culture=neutral, PublicKeyToken=53879b3e20e7a7d6'. El sistema no puede encontrar el archivo espec
do.
   at Helper.MultiespectralToTiffHelper.Save(Byte[] multiespectral, String path, IDictionary`2 properties)
   at tifflibnet.Program.Main(String[] args) in C:\Users\arturo\Documents\Desarrollo\tifflibnet\Program.cs:line 32`

**Comment by sygfk** [comment]
I have the same problem on windows 10 even downgrade to version 2.4.582. It compiles ok, but this error were throw out at run time. 

Error message
An unhandled exception of type 'System.IO.FileNotFoundException' occurred in mscorlib.dll
Additional information: Could not load file or assembly 'BitMiracle.LibTiff.NET40, Version=2.4.582.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.

**Comment by t246246** [comment]
Same problem with Windows 7. Visual Studio professional 2017 15.3.4 with .net 4.7.

**Comment by roboteck** [comment]
Same problem with Windows 10 under Visual Studio community 2017

**Comment by shibaev** [comment]
Reproduce the issue. Will research and fix in the near time.

**Comment by shibaev** [comment]
Fixed in 2.4.584.1, there were problem in our scripts to publish NuGet package.
https://www.nuget.org/packages/BitMiracle.LibTiff.NET/2.4.584.1

Please test and let me know your feedback.

**Comment by arturocandela** [comment]
I've testested the library in a .NETStandard Library project in VisualStudio 2017 Community (15.3.4) and worked as expected
